SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/GridBasedLBStrategy.h

```diff
@@ -48,6 +48,9 @@ class SPATIALGDK_API UGridBasedLBStrategy : public UAbstractLBStrategy
 	virtual bool RequiresHandoverData() const override { return Rows * Cols > 1; }
 
 	virtual FVector GetWorkerEntityPosition() const override;
+
+	virtual uint32 GetMinimumRequiredWorkers() const;
```

Comment:
Nit: I think we add override as a convention

Reply:
good catch!